### PR TITLE
Tests: Do not require venv

### PR DESCRIPTION
**Pull request by xeruf**
Improves situation regarding #94 

Also utilizes basic shell tools to simplify the scripts

### CODE REVIEW

**Comment by xeruf** [comment]
by the by, I recommend checking out https://cbea.ms/git-commit/ :)

**Comment by novoid** [comment]
Thanks for the improvement - will read the commit message recommendations :-)

File: start_end-to-end-test.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ end_to_end_test()
 
 }
 
-end_to_end_test "$@"
+end_to_end_test "$@" && \
 echo "\n\n           End-to-end tests ended successfully! :-)\n\n"
```
Comment (xeruf) [inline start_end-to-end-test.sh:43]: this is not needed because the script has the `errexit` (-e) flag set

Comment (novoid) [reply start_end-to-end-test.sh:43]: Oh yes, that's redundant then.

**Pull request merged**